bin/wm_apply_ORG_atlas_to_subject.sh

```diff
@@ -389,22 +389,30 @@ fi
 
 if [ $DiffMeasure == 1 ]; then
 
+  os=$(uname)
+
+  if  [[ "$os" == 'Linux' ]]; then
+    measurement_cli_cmd=$SlicerPath" --launch "$FiberTractMeasurementsCLI
+  else
+    measurement_cli_cmd=$FiberTractMeasurementsCLI
+  fi
+
 	echo "<wm_apply_ORG_atlas_to_subject> Report diffusion measurements of fiber clusters."
```

Comment:
The indentation here has used a tab instead of whitespaces, hence the difference with the newly added code.